frontend/src/components/SearchModal.tsx

```
@@ -0,0 +1,42 @@
+            <h2 className="text-white text-lg font-semibold">Search Contacts</h2>
+            <button onClick={onClose} className="text-gray-400 hover:text-white">
+              <X className="h-5 w-5" />
+            </button>
+          </div>
+          <div className="relative">
+            <Search className="absolute left-3 top-1/2 -translate-y-1/2 size-4 text-gray-400" />
+            <input
+              type="text"
+              placeholder="Search contacts..."
+              className="w-full bg-[#171A21] border-2 border-zinc-800 rounded-xl py-2 pl-10 pr-4 text-white placeholder:text-gray-500 focus:outline-none focus:border-emerald-400 transition-colors"
+              autoFocus
```

Biome (2.4.6)

[error] Avoid the autoFocus attribute.

(lint/a11y/noAutofocus)